paddle/fluid/inference/api/analysis_predictor.cc

```diff
@@ -900,6 +900,15 @@ void AnalysisPredictor::OptimizeInferencePirProgram() {
           }
         }
       }
+      if (config_.mkldnn_bfloat16_enabled()) {
```

Comment:
This code should be put under 
`if (!config_.custom_pass_only_) {`
Since custom pass has the highest priority if enabled, and oneDNN pass should be activated without it.